packages/file-export/fastfetchbot_file_export/video_download.py

```
@@ -0,0 +1,210 @@
+        return "bv*[height<=480]+ba/b[height<=480] / wv*+ba/w"
+    raise ValueError("no available extractor found")
+
+
+def init_yt_downloader(
+    hd: bool = False,
+    audio_only: bool = False,
+    extractor: str = None,
+    no_proxy: bool = False,
+    extract_only: bool = False,
+    video_format: str = None,
+    download_dir: str = "/tmp",
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)